### PR TITLE
Tim/514-216-build-the-development-console

### PR DESCRIPTION
**Pull request by tg339**
added the moose console and up

### CODE REVIEW

File: apps/moose-console/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Getting Started
+
+First, run the development server:
+
+```bash
+yarn dev
```
Comment (callicles) [inline apps/moose-console/README.md]: `yarn`?

Should we remove this altogether or fix it

File: apps/moose-console/sentry.server.config.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// This file configures the initialization of Sentry on the server.
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "https://ad3e8eb464c321b18b78c8402e80a8e5@o4505851966128128.ingest.sentry.io/4505852583084032",
```
Comment (callicles) [inline apps/moose-console/sentry.server.config.ts]: Did you update those values for with a new entry for the console in sentry?

**Comment by callicles** [comment]
You have a linting thing to fix for the tests to pass. ONce we have addressed the comments we probably should merge quickly

**Review by callicles — COMMENTED** [review]
Mostly looks good to me

File: apps/moose-console/sentry.server.config.ts (hunk 2 above)
Comment (tg339) [reply apps/moose-console/sentry.server.config.ts]: No :( )

File: apps/moose-console/README.md (hunk 1 above)
Comment (tg339) [reply apps/moose-console/README.md]: probably remove

**Review by callicles — APPROVED** [review]
LGtM

**Pull request merged**